client.go

```diff
@@ -246,24 +264,28 @@ func (client *NativeClient) Shell(args ...string) error {
 		ssh.ECHO: 1,
 	}
 
-	fd := os.Stdin.Fd()
+	fd := int(os.Stdin.Fd())
 
-	if term.IsTerminal(fd) {
```

Comment:
I want to say there was some reason we were using docker's `term` but I'll dig in and look at this again.